### PR TITLE
vendor: update continuity to 0377f7d767206

### PR DESCRIPTION
**Pull request by kolyshkin**
This is to include the Go 1.11 fix (https://github.com/containerd/continuity/pull/120). Originally from https://github.com/moby/moby/pull/37358.

full diff: https://github.com/containerd/continuity/compare/d3c23511c1bf5851696cba83143d9cbcd666869b...0377f7d767206f3a9e8881d0f02267b0d89c7a62

Also includes: https://github.com/containerd/continuity/pull/111 "Support cancellation via context in DiskUsage"

### CODE REVIEW

File: vendor/archive/tar/LICENSE [hunk 1]
```diff
@@ -1,27 +0,0 @@
-Copyright (c) 2009 The Go Authors. All rights reserved.
-
-Redistribution and use in source and binary forms, with or without
-modification, are permitted provided that the following conditions are
-met:
```
Comment (thaJeztah) [inline vendor/archive/tar/LICENSE]: whoops; looks like you accidentally removed this vendor as well

**Review by thaJeztah — CHANGES_REQUESTED** [review]
LGTM after the archive vendoring is restored

File: vendor/archive/tar/LICENSE (hunk 1 above)
Comment (kolyshkin) [reply vendor/archive/tar/LICENSE]: I seem to always have troubles with vndr :(

Comment (kolyshkin) [reply vendor/archive/tar/LICENSE]: fixed

**Review by thaJeztah — APPROVED** [review]
LGTM, thanks!

**Comment by thaJeztah** [comment]
failures look to be flaky tests


ping @vdemeester PTAL

**Review by vdemeester — APPROVED** [review]
LGTM 🦁

**Pull request merged**